react-with-graphql-tutorial.md

```diff
@@ -745,7 +746,7 @@ const GET_ORGANIZATION = `
 `;
 {{< /highlight >}}
 
-You can use template literals in JavaScript to define the query as string with multiple lines. It should be identical to the query that you have used in GraphiQL before. Now, you can use axios to make a POST request to GitHub's GraphiQL API. The configuration for axios already points to the correct API endpoint and uses your personal access token. The only thing left is passing the query to it as payload when doing a POST request. The argument for the endpoint can be an empty string, because you have defined the endpoint before in the configuration. You are going to execute the request when the App component mounted in `componentDidMount()`. After the promise from axios has been resolved, you will only console log the result.
+Use template literals in JavaScript to define the query as string with multiple lines. It should be identical to the query you used before. Now, you can use axios to make a POST request to GitHub's GraphiQL API. The configuration for axios already points to the correct API endpoint and uses your personal access token. The only thing left is passing the query to it as payload during a POST request. The argument for the endpoint can be an empty string, because you defined the endpoint in the configuration. It will execute the request when the App component mounts in `componentDidMount()`. After the promise from axios has been resolved, only a console log of the result remains.
```

Comment:
"It should be identical to the query you used before." to "It should be identical to the query you used before in GraphiQL or GitHub Explorer."